CPT331.WebAPI/Validation/ValidateCoordinatesAttribute.cs

```diff
@@ -0,0 +1,86 @@
+﻿#region Using References
+
+using CPT331.Core.ObjectModel;
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Web.Http.Controllers;
+using System.Web.Http.Filters;
+using System.Web.Http.ModelBinding;
+
+#endregion
+
+namespace CPT331.WebAPI.Validation
+{
+	public class ValidateCoordinatesAttribute : ActionFilterAttribute
```

Comment:
Coordinates are validated by this action attribute - requests won't even make it to the controller unless they pass this first.